### ISSUE TITLE
wk1 dec_epic Share Clinic Selections with Other Team Members - Share Button OR Report Bookmarks

### ISSUE THREAD

**Issue by mnallajerla** [lzim/teampsd#2401]
"Share" button on "Report" Bookmark for teams to share Clinic Selections and other edits in PBI app across team

**Comment by mnallajerla** [comment]
Discussed @ 8 AM WG Leads

- Requirement to be tracked as "Share ClinicSelection filters with other team members as 'personal bookmarks'" Card #2401
- Requirement - Cannot turn the "share" button on in the App version.
- But, a new feature is coming to use "Personal Bookmarks" to share selections.


Possible work around until this feature is released:
- Use "pre-defined report bookmarks" to set up selections.

Concern with this workaround:
- People would see that there are reports set up for other sta3n (stations/facilities).


DECISION: We'll pursue this workaround for now, because due to Row Level Security (RLS) people would not be able to see data from other sta3n (possibly not even ClinicSelection tab).
DECISION: Facilitators will set up this report bookmark directly in the App w/team lead.


@lijenn & @mnallajerla  will add this to the

- Data UI Power BI cheatsheet (learner)
AND
- Facilitator 1-pager.

**Key step: Clear out all the selections once done setting up the Report bookmark (or they will be setting up defaults for everyone).**

@jeffhoerle  - Admin will look into the build permissions and implications of Facilitators' "Admin" access.

Which permissions go with each of these?
Member
Contributor
Viewer

**Comment by mnallajerla** [comment]
We have a way to create and share report bookmark. Please see the data cheatsheet for instructions. https://github.com/lzim/mtl/blob/master/blue/session02/s02_learner/mtl_how_2.5_data_cheatsheet.pdf

**Comment by jeffhoerle** [comment]
Update: We abandoned the use of Report Bookmarks.  We found that it would create an unsustainable dimension of our DEV > TEST > PROD workflow.

Problematic Use Case:
1 - Programmer publishes new version of VISN 22 Data UI to cloud
2 - Facilitator creates new Report Bookmark ("Chula Vista 20220101")
3 - Programmer discovers bug and downloads the latest VISN 22 Data UI from cloud
4 - Facilitator edits existing Report Bookmark ("Chula Vista 20220101")
5 - Programmer fixes bug, but can not publish to cloud until manually editing Report Bookmark ("Chula Vista 20220101")

Report Bookmarks, would require a manual review (and edits) of ALL report bookmarks by the programmer to all 17 VISN reports.  This would add a lot of work, and would make impossible to establish an efficient workflow.
